.github/renovate.json

```diff
@@ -1,16 +1,15 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
 	"automerge": true,
-	"extends": ["config:best-practices", "replacements:all"],
+	"extends": [
+		":preserveSemverRanges",
+		"config:best-practices",
+		"replacements:all"
+	],
 	"ignoreDeps": ["codecov/codecov-action"],
 	"labels": ["dependencies"],
 	"minimumReleaseAge": "7 days",
-	"packageRules": [
-		{
-			"enabled": false,
-			"matchDepNames": ["debug"]
-		}
-	],
```

Comment:
🤔 I think I'm good to remove this. `debug` is listed in `dependencies`, so I'd hope Renovate doesn't change what end-users see.